docs/developer-docs/smart-contracts/call/arguments.mdx

```diff
@@ -184,40 +194,14 @@ fn get_text(d: Day) -> &'static str {
 <AdornedTab value={"typescript"} label="TypeScript" endAdornment={<BetaChip />}>
 
 ```typescript
-import { Canister,  query, Variant } from 'azle';
-
-const Day = Variant({
-    Sun;
-    Mon;
-    Tue;
-    Wed;
-    Thu;
-    Fri;
-    Sat;
+export const ReactionType = IDL.Variant({
+    Fire: IDL.Null,
+    ThumbsUp: IDL.Null,
+    ThumbsDown: IDL.Null
 });
 
-export default Canister({
-    get_text: query([Day], Day, (d)) => {
-        switch (d) {
-            case Day.Sun:
-                return "Sunday";
-            case Day.Mon:
-                return "Monday";
-            case Day.Tue:
-                return "Tuesday";
-            case Day.Wed:
-                return "Wednesday";
-            case Day.Thu:
-                return "Thursday";
-            case Day.Fri:
-                return "Friday";
-            case Day.Sat:
-                return "Saturday";
-    }
-}
-});
 ```
-To learn more about variants in Typescript via Azle, refer to [the Azle book reference on variants](https://demergent-labs.github.io/azle/reference/candid/variant.html?highlight=variant#variant).
+To learn more about variants in Typescript via Azle, refer to [the Azle code](https://github.com/demergent-labs/azle/blob/main/tests/end_to_end/candid_rpc/class_syntax/complex_types/src/candid_types.ts#L3).
```

Comment:
I generally spell `Typescript` `TypeScript` and it's spelled that way elsewhere in this documentation, I think that's the best way to subjectively go.

Reply:
fixed capitalization